src/Orchard.Web/Modules/Orchard.Projections/Migrations.cs

```diff
@@ -282,5 +281,38 @@ public class Migrations : DataMigrationImpl {
 
             return 4;
         }
+        public int UpdateFrom4() {
+            SchemaBuilder.AlterTable("StringFieldIndexRecord", table => table
+            .AddColumn<string>("LatestValue", c => c.WithLength(4000)));
+
+            SchemaBuilder.AlterTable("IntegerFieldIndexRecord", table => table
+            .AddColumn<long>("LatestValue"));
+
+            SchemaBuilder.AlterTable("DoubleFieldIndexRecord", table => table
+            .AddColumn<double>("LatestValue"));
+
+            SchemaBuilder.AlterTable("DecimalFieldIndexRecord", table => table
+            .AddColumn<decimal>("LatestValue"));
+
+            //Adds indexes for better performances in queries
```

Comment:
Two comments:
- We need to talk about the "necessity" of these indices
- You are reusing the same index name multiple times, is it an issue? should it be unique per database, or no otherwise table prefix would break it?

Reply:
- Necessity of these indexes: do you mean SQL indexes or the "LatestValue" columns? For SQL indexes, I think at least `IX_FieldIndexPartRecord_Id` should be created because it is a foreign key "in fact". The index over PropertyName column is for projections and HqlQueries so I think they should be there. I don't see a good reason why they shouldn't be there
- Same index name: I seem to remember that Indexes name should be unique per table and not per database. At least in SQL Server, so i don't see an issue here. Perhaps it's not the same in MySql or Postgre?